cluster/node-pools/master-default/userdata.yaml

```diff
@@ -580,7 +580,7 @@ write_files:
           effect: NoSchedule
         containers:
         - name: kube-controller-manager
-          image: nonexistent.zalan.do/teapot/kube-controller-manager:fixed
+          image: {{if eq .Cluster.ConfigItems.kubernetes_controller_manager_image "zalando" }}registry.opensource.zalan.do/teapot/kube-controller-manager-internal:v1.21.9-2{{else}}nonexistent.zalan.do/teapot/kube-controller-manager:fixed{{end}}
```

Comment:
This should follow the same approach as for `kube-scheduler`, shouldn't it?

Reply:
* https://github.bus.zalan.do/teapot/kubernetes-on-ubuntu/pull/221
* https://github.com/zalando-incubator/kubernetes-on-aws/pull/5055

Reply:
Yes, this was only to enable quicker roll out of just the controller-manager changes. The idea was to bring it back once we also update the AMI.